spacepy/pybats/__init__.py

```diff
@@ -115,10 +115,12 @@ class methods with the *calc_* prefix and call them.
 # Global imports (used ubiquitously throughout this module.
 import os.path
 from functools import wraps
+from turtle import color
```

Comment:
As far as I can tell, this import isn't used. If it is, the "from" import really obscures its origin. Should probably either import turtle so we have the namespacing, or remove.

Reply:
This snuck in thanks to VSCode's rather aggressive auto-complete and was missed by me. Fixed.